bundlewatch.config.js

```diff
@@ -12,7 +12,7 @@ const getMaxSizeForPackage = packageLocation => {
 
 const getChangedPackageLocations = () => {
     let outputPackages;
-    let command = process.env.CIRCLE_BRANCH === 'master' || process.env.RUN_ALL ? "npx lerna ls --json" : "npx lerna ls --since origin/master --json" 
+    let command = process.env.CIRCLE_BRANCH === 'master' || process.env.RUN_ALL === 'true' ? "npx lerna ls --json" : "npx lerna ls --since origin/master --json" 
```

Comment:
for my own understanding - is this simply because the boolean config value is represented as a string?